src/handlers/parseRawDependencyDataHandler.ts

```diff
@@ -31,7 +32,12 @@ export async function parseRawDependencyDataHandler(project: MavenProject): Prom
     const eol: string = "\r\n";
     const prefix: string = "+- ";
     const [treeNodes, conflictNodes] = await parseTreeNodes(treeContent, eol, indent, prefix, project.pomPath);
+    const dependenciesMenu: DependenciesMenu = <DependenciesMenu> project.getChildren()[2];
```

Comment:
this is tricky to rely on the order. it's not robust if more menus are added under project node.